guess.js

```diff
@@ -1,12 +1,14 @@
 //setting name game 
 let nameGame = "guess The Word";
+// [REVIEW] Why are you setting these HTML elements using JavaScript?
 document.title = nameGame;
 document.querySelector("h1").innerHTML = nameGame;
 document.querySelector("footer").innerHTML = `${nameGame} Game created by Mohamed Sabry`
 
 // setting Game options
 
 const numbersOfTries = 6;
+// [REVIEW] Instead of hardcoding the number of letters, can it be derived from the actual word length?

```

Comment:
you mean i can yuse worGuess.length instade of numbersOfletters

Reply:
Yes exactly.